lib/view_component/instrumentation.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require "active_support/notifications"
+
+module ViewComponent # :nodoc:
+  module Instrumentation
+    def self.included(mod)
+      mod.prepend(self)
+    end
+
+    # Configure if ActiveSupport notifications are enabled for all ViewComponents.
+    mattr_accessor :instrumentation_enabled, instance_writer: false, default: false
```

Comment:
Is this still being used? It looks like we're checking `Rails.application.config` elsewhere.